### DISCUSSION TITLE
Is there some changes of `SqlxMySqlConnector::from_sqlx_mysql_pool`?

### DISCUSSION THREAD

**Discussion by lz1998** [SeaQL/sea-orm#375] [Q&A] [answered]
Is there some changes of `SqlxMySqlConnector::from_sqlx_mysql_pool`?

I can run this several days ago, but today I execute `cargo clean` and build again, an error orrurs.

Cargo.toml
```toml
sea-orm = { version = "0.4.0", git = "https://github.com/SeaQL/sea-orm.git", branch = "issues/344", features = ["sqlx-mysql", "runtime-tokio-native-tls", "macros", "with-chrono"], default-features = false }
```

```bash
error[E0308]: mismatched types
 --> src/database/sea.rs:8:46
  |
8 |     SqlxMySqlConnector::from_sqlx_mysql_pool(pool)
  |                                              ^^^^ expected struct `sqlx_core::pool::Pool`, found struct `Pool`
  |
  = note: expected struct `sqlx_core::pool::Pool<sqlx_core::mysql::database::MySql>`
             found struct `Pool<byted_sqlx::sqlx::MySql>`
  = note: perhaps two different versions of crate `sqlx_core` are being used?

For more information about this error, try `rustc --explain E0308`.

```

**Comment by billy1624** [comment, marked as answer]
Hey @lz1998, I guess you are using two different versions of SQLx in your workspace?

**Reply by lz1998** [reply]
> 

I found this is caused by branch "issues/344" using a forked sqlx branch.

I am using a version of sqlx in our company which can connect database without password, but it will return a sqlx pool.

**Reply by billy1624** [reply]
That will cause the issue. Maybe you can update your toml to be something like
```toml
sea-orm = { git = "...", rev = "ff4e00ff6bc599319d3360355a078727fea204e1", ...}
```
Which reference this commit https://github.com/SeaQL/sea-orm/pull/345/commits/ff4e00ff6bc599319d3360355a078727fea204e1
